reference/bundles/custom_entity_bundle/crud_interfaces.rst.inc

```diff
@@ -0,0 +1,298 @@
+CRUD Configuration
+******************
+
+The configuration for the CRUD actions of your custom entities must be in a file named ``config/custom_entities.yml``,
+located in an enabled bundle. To have a full working CRUD for an entity, the following configuration could be used:
+
+.. code-block:: yaml
+
+    # Resources/config/custom_entities.yml
+    custom_entities:
+        color:
+            entity_class: Acme\Bundle\AppBundle\Entity\MyEntity
+            options:
+                acl_prefix: acme_enrich_my_entity
+            actions:
+                edit:
+                    form_type: acme_enrich_my_entity
+                create:
+                    form_type: acme_enrich_my_entity
+
+
+The root level of the file contains the configuration for all your entities, indexed by alias. The alias will be used in the
+CRUD URLs, and later, for the datagrid configuration.
+
+For each entity, the following options are available:
+
+- **abstract**: set to `true` if the definition is only meant to be extended
+- **extends**: the alias of the extended configuration. The bundle offers three base configurations that can be extended: default, quick_create, and mass_actions
+- **options**: general options for the CRUD
+- **actions**: the configuration for the enabled CRUD actions
+- **entity_class**: the class of the entity, **required** if the configuration is not abstract.
+  (Container parameters can be used in the class value)
```

Comment:
This render pretty well 

![render](https://cloud.githubusercontent.com/assets/1247388/23364022/a265df30-fcfd-11e6-84f8-bf28e9b020a0.PNG)